package.json

```diff
@@ -53,15 +53,14 @@
     "grunt-coveralls": "^1.0.1",
     "grunt-fh-build": "2.0.0",
     "istanbul": "^0.4.3",
-    "mocha": "^2.2.5",
+    "mocha": "^5.1.1",
     "nock": "^8.0.0",
     "proxyquire": "^1.7.4",
     "recursive-readdir": "^2.0.0",
     "sinon": "^1.17.3",
-    "sinon-as-promised": "4.0.2",
-    "turbo-test-runner": "0.3.3"
```

Comment:
Why remove turbo-test-runner ?

Reply:
The turbo should not so long be used as it is deprecated and we should use the mocha. The test is using _mocha and not turbo anyway.